README.md

```diff
@@ -27,6 +27,16 @@ This project is currently in an experimental state. All functionality and naming
 
 This project publishes multiple packages and is managed using [pnpm](https://pnpm.io/)
 
+### @sap-ai-sdk/orchestration
+
+This package incorporates generative AI orchestration capabilities into your AI activities in SAP AI Core and SAP AI Launchpad.
```

Comment:
In case we haven't linked it elsewhere already:

```suggestion
This package incorporates generative AI [orchestration](https://help.sap.com/docs/sap-ai-core/sap-ai-core-service-guide/orchestration) capabilities into your AI activities in SAP AI Core and SAP AI Launchpad.
```

Reply:
I have linked it in the orchestration client documentation, doesn't hurt to add it here too.